packages/ui/react-ui-theme/src/config/tokens/sememes-system.ts

```diff
@@ -74,6 +75,10 @@ export const systemSememes = {
     light: ['neutral', surfaceCadence.light[4]],
     dark: ['neutral', surfaceCadence.dark[6]],
   },
+  blocked: {
+    light: ['amber', 500],
+    dark: ['amber', 500],
+  },
```

Comment:
Excellent ✨

Reply:
Maybe call this `warning`, since we also use amber at these luminosities for this message valence (been meaning to replace all the tokens in messageValence.ts at some point).